pv_checks.yml

```diff
@@ -0,0 +1,160 @@
+---
```

Comment:
seeing as we are specifying the pvs required as a prereq then do we need to run the pv_checks.yml here?